sale_order_invoicing_queued/readme/INSTALL.rst

```diff
@@ -1,2 +1,5 @@
 This module depends on *queue_job* module that is hosted on
 https://github.com/OCA/queue.
+
+It only works in an updated 13.0 version after the 4th of May of 2020,
```

Comment:
```suggestion
It only works in an updated 13.0 version after the 4th of March of 2020,
```

Reply:
Hahaha, it would be impossible to be used until 2 months in the futur :stuck_out_tongue_closed_eyes:

Reply:
Fixed!